test/e2e/input-json5/actor/package.json

```diff
@@ -9,6 +9,7 @@
     "overrides": {
         "apify": {
             "@crawlee/core": "file:./packages/core",
+            "@crawlee/types": "file:./packages/types",
```

Comment:
I noticed this is missing in all other tests. Should that be a separate PR?

Reply:
it probably does not matter at all, as its a type only package (and most of the tests dont use TS at all).

no need for more PRs, this is a chore/test type of commit, it should not be part of changelog. i would rather see single PR that actually fixes this, i dont mind more smaller PR if they deliver things that should result in separate changelog lines, otherwise its not that important.